test/e2e/metrics_util.go

```diff
@@ -284,6 +284,8 @@ func HighLatencyRequests(c *client.Client) (int, error) {
 	}
 
 	Logf("API calls latencies: %s", prettyPrintJSON(metrics))
+	// Log perf data
```

Comment:
Can you please add a TODO to get rid of the previous line when we migrate to the new form of perf-dash?